lib/rpm_package.py

```diff
@@ -161,6 +161,12 @@ def _load(self):
             self.download_build_files = files.get('download_build_files', [])
 
             # list of dependencies
+            for dep_name in files.get('runtime_dependencies', []):
+                dep = RPM_Package.get_instance(dep_name, self.distro)
+                self.dependencies.append(dep)
```

Comment:
Since you're updating the yaml attribute, it would be better to rename the variable accordingly to avoid ambiguity, don't you think?
Like I mentioned in https://github.com/open-power-host-os/versions/pull/118#issuecomment-277278343, I prefer the name `install_dependencies`.